src/main/asciidoc/types/index.adoc

```diff
@@ -61,16 +61,15 @@ The Neo4j driver maps Neo4j types to native language types as follows:
 | Neo4j        | Java
 | Null         | null
 | Boolean      | java.lang.Boolean
-| Integer      | java.lang.Integer
+| Integer      | java.lang.Long
 | Float        | java.lang.Double
 | String       | java.lang.String
 | List         | java.util.List<T>
-| Map          | java.util.Map<String, Value> (*)
+| Map          | java.util.Map<TKey, TValue>
```

Comment:
I changed to generic types and we could use whatever names there. But the java doc prefer [`Map<K, V>`] (https://docs.oracle.com/javase/7/docs/api/java/util/Map.html), so change it to be more java